cmd/worker.go

```diff
@@ -16,16 +16,22 @@ import (
 const (
 	WorkerAsyncBlockHasherMaxBlockSizeFlag = "worker-async-block-hasher-max-block-size"
 	WorkerAsyncBlockHasherScheduleFlag     = "worker-async-block-hasher-schedule"
+	WorkerBucketDeletionScheduleFlag       = "worker-bucket-deletion-schedule"
+	WorkerBucketDeletionGraceDaysFlag      = "worker-bucket-deletion-grace-days"
 )
 
 type WorkerConfiguration struct {
-	HashLogsBlockMaxSize  int    `mapstructure:"worker-async-block-hasher-max-block-size"`
-	HashLogsBlockCRONSpec string `mapstructure:"worker-async-block-hasher-schedule"`
+	HashLogsBlockMaxSize      int    `mapstructure:"worker-async-block-hasher-max-block-size"`
+	HashLogsBlockCRONSpec     string `mapstructure:"worker-async-block-hasher-schedule"`
+	BucketDeletionCRONSpec    string `mapstructure:"worker-bucket-deletion-schedule"`
+	BucketDeletionGraceDays   int    `mapstructure:"worker-bucket-deletion-grace-days"`
 }
 
 func addWorkerFlags(cmd *cobra.Command) {
 	cmd.Flags().Int(WorkerAsyncBlockHasherMaxBlockSizeFlag, 1000, "Max block size")
 	cmd.Flags().String(WorkerAsyncBlockHasherScheduleFlag, "0 * * * * *", "Schedule")
+	cmd.Flags().String(WorkerBucketDeletionScheduleFlag, "0 0 0 * * *", "Schedule for bucket deletion (default: daily at midnight)")
```

Comment:
Is it a good policy? This wasn't explicitly stated in the specs, was it prompted explicitly?

Reply:
The goal is to have a two-step deletion of buckets: first, the bucket disappears from the API, but the data remains; second, we actually delete the data. There are several days between the two operations.